fgpyo/fasta/builder.py

```diff
@@ -9,33 +9,39 @@
 Writing a FASTA with two contigs each with 100 bases:
 
 ```python
-    >>> from fgpyo.fasta.builder import FastaBuilder
-    >>> builder = FastaBuilder()
-    >>> builder.add("chr10").add("AAAAAAAAAA", 10)
-    >>> builder.add("chr11").add("GGGGGGGGGG", 10)
-    >>> builder.to_file(path = pathlib.Path("test.fasta"))
+>>> import pathlib
```

Comment:
```suggestion
>>> from pathlib import Path
```

Is this correct? The rest of the block references `Path` but not `pathlib`